src/Controller/EsinController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\EsinType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Model\Upload;
+use App\Entity\Esin;
+use App\Service\DataRecovery;
+use App\Repository\EsinRepository;
+
+/**
+ * @Route("/esin")
+ */
+class EsinController extends AbstractController
+{
+
+   /**
+    * @Route("/", name="esin_index", methods={"GET"})
+    */
+    public function index(EsinRepository $esinRepository): Response
+    {
+        return $this->render('esin/index.html.twig', [
+            'esins' => $esinRepository->findAll(),
+        ]);
+    }
+
+
+   /**
+    * @Route("/show", name="esin_show")
+    */
+
+    public function show(EsinRepository $esinRepository): Response
+    {
+        return $this->render('esin/show.html.twig', [
+            'esins' => $esinRepository->findAll(),
+        ]);
+    }
```

Comment:
index and show seems to be exactly the same method except for the template.
If it's not unnecessary duplicated code, we need a comment above each method to explain it's purpose.